cfme/common/provider.py

```diff
@@ -75,6 +88,7 @@ def __init__(self, **kwargs):
             self.candu = kwargs.get('candu')
             self.domain = kwargs.get('domain')
             self.ssh = kwargs.get('ssh')
+            self.token = kwargs.get('token')
```

Comment:
As we're in here, I've always hated this, do you think we could make it a ```credential.type``` attribute?